lab-jeff/readme.md

```diff
@@ -0,0 +1,5 @@
+**A terminal with 'node' and 'npm' is required to run this program.**
+
+All dependencies can be installed by running the command 'npm init' from your terminal from the directory 'lab-jeff'.
```

Comment:
Close. It's actually 'npm install' or 'npm i' that you want here. If a package.json already exists for your project (which it does because you created it), then the next person who wants to use your app just needs to install the dependencies that you defined in package.json. 'npm init' is used when a package.json needs to be created.